jupyterhub/environment.yaml

```diff
@@ -14,4 +14,4 @@ dependencies:
   - pip:
     - nebari-jupyterhub-theme
     - python-keycloak==0.26.1
-    - jhub-apps==2024.1.2
+    - jhub-apps==2024.1.2
```

Comment:
We should actually move this to conda install instead of pip to have more reliable/reproducible solves.